src/Raw/index.js

```diff
@@ -55,6 +55,19 @@ Raw.boolean = function (input) {
 }
 
 /**
+ * @description tells whether input is equal the other input
+ * boolean or not
+ * @method boolean
+ * @param  {String} input
+ * @param  {String} second_input
+ * @return {Boolean}
+ * @public
+ */
+Raw.equal = function (input, second_input) {
```

Comment:
There is a method called `Raw.same`. You can use that https://github.com/poppinss/indicative/blob/develop/src/Raw/index.js#L412